data-engineer-01.md

```diff
@@ -9,19 +9,20 @@ Help us improve our data stack which allows us to process millions of data point
 * You'll be responsible ingesting all necessary business data into our data lake and enhance our Extract-Load processes
 * Facilitate growth of our event collection platform by collaborating with stakeholders effectively
 * Be the safety net for the team when it comes to instrumenting pipelines with meaningful event logs & alerts
-* Proactively push your ideas based on your collaboration with Data Analysts & ML Engineers
+* Proactively push your ideas based on your collaboration with Data & Business Analysts
 * Be a key driver for enhancing our existing stack by testing and applying new techniques and tools
 * Be part of a self-driven team that is open for opportunities
 
 ## Your Profile
 
+* You have 4+ years of experience working as a Software Engineer or Data Engineer
```

Comment:
I think makes sense, it can capture Software Engineers that want to migrate to Data Engineer roles